.github/workflows/ci.yml

```diff
@@ -26,21 +26,10 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Load pip cache
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/test-requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
-
-      - name: Install dependencies
-        run: |
-          pip install --upgrade pip
-          pip install -r test-requirements.txt
-
       - name: Test typing_extensions
-        run: pytest typing_extensions/src_py3
+        run: |
+          cd typing_extensions/src_py3
```

Comment:
```suggestion
        run: |
          # Be wary of running `pip install` here, since it becomes easy for us to
          # accidentally pick up typing_extensions as installed by a dependency 
          cd typing_extensions/src_py3
```